diagnostics-app/src/app/app.component.ts

```diff
@@ -8,20 +8,12 @@
  */
 
 import { Component } from '@angular/core';
-import { SwUpdate } from '@angular/service-worker';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css'],
 })
 export class AppComponent {
-  constructor(private updates: SwUpdate) {
-    updates.available.subscribe((event) => {
-      updates.activateUpdate().then(() => {
-        document.location.reload();
-        console.log('Updated app to latest version!');
-      });
-    });
-  }
```

Comment:
These lines are responsible for refreshing the installed PWA automatically if it's updated. The `SwUpdate` is difficult to mock while testing and is not needed for now. So I have removed this feature for now.

The user can simply refresh the app and fetch updates, if any.

Reply:
Ack.